src/Models/Hooks/useRuntimeSceneData.ts

```diff
@@ -166,11 +184,44 @@ export const useRuntimeSceneData = (
                 }
             });
 
+            // fetch the config
+            const pollingConfig = ViewerConfigUtility.getPollingConfig(
+                scenesConfig,
+                sceneId
+            );
+
+            const computeInterval = (
+                twinCount: number,
+                pollingConfig: IPollingConfiguration
+            ) => {
+                const MIN_INTERVAL = 10000;
+                const fastestPossibleRefreshRateSeconds = Math.max(
+                    twinCount * 1000, // 1s per twin
+                    MIN_INTERVAL
+                );
+                const actualRefreshRateSeconds =
+                    pollingConfig.pollingStrategy === 'Limited' &&
```

Comment:
ahh you already defined a type for it, lets use it
```suggestion
                    pollingConfig.pollingStrategy === IPollingStrategy.Limited &&
```